docs/snippets/svelte/button-story-with-args.ts-4-9.mdx

```diff
@@ -0,0 +1,35 @@
+```ts
+// Button.stories.ts
+
+import type { Meta, StoryObj } from '@storybook/svelte';
+
+import Button from './Button.svelte';
+
+const meta = {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/svelte/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'Button',
+  component: Button,
+} satisfies Meta<typeof Button>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/svelte/api/csf
+ * to learn how to use render functions.
+ */
+export const Primary: Story = {
+  render: (args) => ({
+    Component: Button,
+    props: args,
+  }),
```

Comment:
@miily8310s, we can drop the render function and comment in this example and the pre-TS variation with the same name.